source/_integrations/aws_s3.markdown

```diff
@@ -122,6 +124,41 @@ The AWS S3 integration has the following limitations:
 
 This integration is designed to work only with the official Amazon AWS S3 service. Despite claims of S3 API compatibility, third-party storage providers like Wasabi, DigitalOcean Spaces, Backblaze B2, Infomaniak, OVH Cloud, and others have often proven to be incompatible. Even when they appear to work initially, they cannot guarantee ongoing compatibility with this AWS S3 integration in the future.
 
+## Troubleshooting
+
+{% details "Backups appear under the wrong Home Assistant instance" %}
+
+If backups are displayed in the wrong Home Assistant instance or assigned to the wrong S3 configuration in the backups menu, the prefix could overlap with the prefix of another Home Assistant instance.  
+For example, instance B may show backups that actually belong to instance A.
+
+**Cause**
+
+Amazon S3 uses **prefix-based paths** to organize objects but does not enforce a real directory structure. If multiple Home Assistant instances or configurations share the same bucket and use **overlapping prefixes**, their backup paths overlap.
+
+**Problematic prefix structure:**
+
+- Instance A: `homeassistant/prod/`  
+- Instance B: `homeassistant/test/`  
+- Instance C: `homeassistant/`
+
+Since `homeassistant/` includes everything below it (like `prod/` and `test/`), backups stored under those sub-prefixes will also appear under the broader prefix.  
+This causes backups to be displayed or assigned incorrectly in the UI, even though they are stored correctly in S3.
+
+**Solution**
+
+Use **unique, non-overlapping prefixes** for each instance or configuration.  
```

Comment:
```suggestion
Use *unique, non-overlapping prefixes* for each instance or configuration.  
```
bold is used to mark UI strings. we try to avoid using it to hightlight text.